wkcuber/README.md

```diff
@@ -1,155 +1,49 @@
-
-The tools are modular components to allow easy integration into existing pipelines and workflows.
+Python CLI for creating and working with [WEBKNOSSOS](https://webknossos.org/) [WKW](https://github.com/scalableminds/webknossos-wrap) datasets. WKW is a container format for efficiently storing large, scale 3D image data as found in (electron) microscopy.
 
 ## Features
 
-* `wkcuber`: Convert supported input files to fully ready WKW datasets (includes type detection, downsampling, compressing and metadata generation)
-* `wkcuber.convert_image_stack_to_wkw`: Convert image stacks to fully ready WKW datasets (includes downsampling, compressing and metadata generation)
-* `wkcuber.export_wkw_as_tiff`: Convert WKW datasets to a tiff stack (writing as tiles to a `z/y/x.tiff` folder structure is also supported)
-* `wkcuber.cubing`: Convert image stacks (e.g., `tiff`, `jpg`, `png`, `bmp`, `dm3`, `dm4`) to WKW cubes
-* `wkcuber.tile_cubing`: Convert tiled image stacks (e.g. in `z/y/x.ext` folder structure) to WKW cubes
-* `wkcuber.convert_knossos`: Convert KNOSSOS cubes to WKW cubes
-* `wkcuber.convert_nifti`: Convert NIFTI files to WKW files (Currently without applying transformations).
-* `wkcuber.convert_raw`: Convert RAW binary data (.raw, .vol) files to WKW datasets
-* `wkcuber.downsampling`: Create downsampled magnifications (with `median`, `mode` and linear interpolation modes). Downsampling compresses the new magnifications by default (disable via `--no_compress`).
-* `wkcuber.compress`: Compress WKW cubes for efficient file storage (especially useful for segmentation data)
-* `wkcuber.metadata`: Create (or refresh) metadata (with guessing of most parameters)
-* `wkcuber.recubing`: Read existing WKW cubes in and write them again specifying the WKW file length. Useful when dataset was written e.g. with file length 1.
-* `wkcuber.check_equality`: Compare two WKW datasets to check whether they are equal (e.g., after compressing a dataset, this task can be useful to double-check that the compressed dataset contains the same data).
-* Most modules support multiprocessing
+wkcuber offers some commands to work with WEBKNOSSOS datasets:
+
+- `wkcuber compress`: Compress a WEBKNOSSOS dataset
+- `wkcuber convert`: Convert a image stack to a WEBKNOSSOS dataset
+- `wkcuber download`: Download a WEBKNOSSOS dataset from a remote location
+- `wkcuber downsample`: Downsample a WEBKNOSSOS dataset
+- `wkcuber upload`: Upload a local WEBKNOSSOS dataset to a remote location
+- `wkcuber upsample`: Upsample a WEBKNOSSOS dataset
 
 ## Supported input formats
 
-* Standard image formats, e.g. `tiff`, `jpg`, `png`, `bmp`
-* Proprietary image formats, e.g. `dm3`
-* Tiled image stacks (used for Catmaid)
-* KNOSSOS cubes
-* NIFTI files
-* Raw binary files
+- Standard image formats, e.g. `tiff`, `jpg`, `png`, `bmp`
+- Proprietary image formats, e.g. `dm3`
+- Raw binary files
```

Comment:
I assume even more biology image formats are possible through the `bioformats` library. Perhaps, that require installing additional dependencies. See `--all-deps`(?) install option in the wk-libs. If that works we should list that here as it would double and triple the amount of supported image file formats. Please check.

Reply:
I updated the installation instructions to install webknossos[all] to get support for most image types.